src/assets/i18n/global/en-global.json

```diff
@@ -183,7 +183,21 @@
     "ANYONE" : "Anyone",
     "SOMETHING_WENT_WRONG": "Something went wrong while uploading files, please try again",
     "DEMO":"This is a demo of the Education Portal. Check documentation",
-    "HERE" : "here"
+    "HERE" : "here",
+    "EDIT_TEMPLATE":"Edit Template",
+    "ADVANCE_EDITOR":"Advance Editor",
+    "SAVE_DOCUMENT": "Save document",
+    "HTML":"HTML",
+    "JSON_SCHEMA":"JSON Schema",
+    "DESCRIPTION":"Description",
+    "VERIFIABLE_CREDENTIAL_TEMPLATE":"Verifiable Credential Template",
+    "ADD_TEMPLATE":"Add Template",
+    "ANY_EDIT":"Any edit in QR code",
```

Comment:
@chaitrali-r instead of ANY_EDIT -> ANY_EDIT_OR_CODE. change it (change in html file too)